sdk/rust/.changie.yaml

```diff
@@ -0,0 +1,35 @@
+changesDir: .changes
+unreleasedDir: unreleased
+headerPath: header.tpl.md
+changelogPath: CHANGELOG.md
+versionExt: md
+versionFormat: '## sdk/rust/{{.Version}} - {{.Time.Format "2006-01-02"}}'
+kindFormat: '### {{.Kind}}'
+changeFormat: '- {{.Body}} by @{{.Custom.Author}} in https://github.com/dagger/dagger/pull/{{.Custom.PR}}'
+custom:
+  - key: PR
+    label: GitHub PR
+    type: int
+  - key: Author
+    label: GitHub Author
+    type: string
+kinds:
+  - label: Added
+  - label: Changed
+  - label: Deprecated
+  - label: Fixed
+  - label: Dependencies
+headerFormat: |
+  This SDK uses 🚙 Engine + 🚗 CLI version `{{.Env.ENGINE_VERSION}}`.[See what changed in that release]](https://github.com/dagger/dagger/releases/tag/{{.Env.ENGINE_VERSION}}).
+  ⬢ https://crates.io/crates/dagger-sdk
+  📒 https://docs.rs/dagger-sdk/
```

Comment:
I wanted to add the exact release number but the format is `major.minor.patch` without `v`.
I can check on Changie if it's possible to trim the version number if that's important!

Reply:
It looks to be using gotemplates, so we could probably just strip the first character

Reply:
Yep it work, here's the output with a dry run for version 0.3.1

```
⬢ https://crates.io/crates/dagger-sdk/0.3.1
📒 https://docs.rs/dagger-sdk/0.3.1/dagger-sdk
```

Fixed in [fd76c23](https://github.com/dagger/dagger/pull/5676/commits/fd76c239b7cae1613bfad5ef5e46dc9185039299)